examples/bpi-monolith-test-suite/notes-and-questions.md

```diff
@@ -0,0 +1,39 @@
+# Session 1
+
+## Introduction 
+- At first everything will be mocked
```

Comment:
What is everything?

Reply:
That was just a note from our first mob programming session. Will remove the file.